lib/response.js

```diff
@@ -30,35 +30,36 @@ Response = (function () {
     this.meta('set_job_status', 'in_progress');
     this.meta('set_job_status_message', status);
     if(data) {
-      this._respond(data || true);
+      this._respond(data || {});
     }
   };
   Response.prototype.job_complete = function (status, data) {
     this.meta('set_job_status', 'done');
     this.meta('set_job_status_message', status);
     if(data) {
-      this._respond(data || true);
+      this._respond(data || {});
     }
   };
   Response.prototype.job_fail = function (status, msg, data) {
     this.meta('set_job_status', 'failed');
     this.meta('set_job_status_message', status);
     this.meta('set_job_failure_message', msg);
     if(data) {
-      this._respond(data || true);
+      this._respond(data || {});
```

Comment:
@dboskovic I mean the method name parameter. It's used only in error method, but not job failed